app/assets/stylesheets/steps.scss

```diff
@@ -25,3 +25,7 @@ pre.markdown-example {
 .table-stepnav-rules .checkbox label {
   margin-top: 0;
 }
+
+.broken-links--list {
```

Comment:
Technically this is an Element, not a Modifier, so it should be `.broken-links__list`.

Reply:
👍 good spot